src/ServicePulse.Host/app/modules/monitoring/views/monitored_endpoints.html

```diff
@@ -28,14 +28,7 @@ <h1>Endpoints overview</h1>
                 <div class="row box-header">
                     <div class="col-sm-12 no-side-padding">
                         <div class="tooltip-trigger">
```

Comment:
If you also remove the link inside the tootltip, then remove the class `tooltip-trigger` and use a standard tooltip attribute for the metric explanation.